.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,94 @@
+name: unit tests
+
+on:
+  push: # run on every push or PR to any branch
+  pull_request:
+  schedule: # run automatically on main branch each Tuesday at 11am
+    - cron: "0 16 * * 2"
+
+jobs:
+  python-unit:
+    name: Python unit tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.6, 3.8]
+        solr: [8.6, 6.6]
+        django: [0, 2.2, 3.0, 3.1]
+    # We use service containers to avoid needing to set up a local copy of
+    # mysql or postgres on the test runner instance. This syntax is similar to
+    # the spec of a docker-compose file. For more, see:
+    # https://docs.github.com/en/free-pro-team@latest/actions/guides/about-service-containers
+    services:
+      solr:
+        image: solr:${{ matrix.solr }}
+        ports:
+          - 8983:8983
+        # workaround permission error in docker image
+        volumes:
+          - /tmp:/opt/solr/server/solr/parasolr_test/
```

Comment:
what's in `/tmp` on the runner? curious what this does

Reply:
that was one of my prior attempts that I failed to remove! will do so now